pkg/vulnerability/vulnerability_test.go

```diff
@@ -212,65 +212,6 @@ func TestClient_FillInfo(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
Could you tell me why you removed this test case?

Reply:
Because the `VendorVectors` field has been removed in https://github.com/aquasecurity/trivy-db/pull/71 and I have open an PR for this: https://github.com/aquasecurity/trivy/pull/718

Reply:
I see. I merged the PR. Thanks!